### DISCUSSION TITLE
Is it possible to use LiveCharts2 on Blazer Server App

### DISCUSSION THREAD

**Discussion by LasseL-HBK** [Live-Charts/LiveCharts2#664] [Q&A]
I've searched for countless hours at this point without luck.

Does anybody know whether it's possible to get LiveCharts2 working on a Blazer Server App?

I've tried to convert from Blazer WebAssembly to Blazer Server App but have been unsuccessful.

Thank you in advance!

**Comment by beto-rodriguez** [comment]
The problem on server side is that how do we send an Image from the server to the client? (In an efficient way)

The animations of the library is not only one image, they are actually a collection of images, that change as the time elapses.

![.](https://raw.githubusercontent.com/beto-rodriguez/LiveCharts2/dev/docs/_assets/1.1.flipbook.gif)

I think that the best alternative to use LiveCharts on Blazor server side or even ASP is to create an image on the server then just use a HTML `img` to display this image.

**Reply by LasseL-HBK** [reply]
I might have completely misunderstood works and how this should be done.

My plan was to have a webpage that low-end web enabled monitors/displays can connect to. I was then planning on using Blazer's SignalR to send updates to the graph, which I'd presume would be HTML5 and JS in the case of blazer

The graph luckily only needs to update every minute, so constant updates aren't necessary, tho I do suspect around 1000 data-points visible at one time.
